lib/trento/domain/cluster/cluster.ex

```diff
@@ -241,6 +243,20 @@ defmodule Trento.Domain.Cluster do
     |> Multi.execute(&maybe_emit_cluster_health_changed_event/1)
   end
 
+  def execute(
+        %Cluster{
+          cluster_id: cluster_id
+        } = cluster,
+        %CompleteChecksExecutionWanda{
```

Comment:
@nelsonkopliku I think this is a good case where common sense can help us instead, I like very much introducing a new command just for the time being and phasing that out as things evolve. I think that commands are really ephemeral and I wouldn't feel comfortable adapting something old to a new usecase. But I mean, by the time we end this discussion, we'll have a newer solution knocking at the door :smile:

Reply:
For me it was the easiest way of tracking the transition and knowing in the future, what we can safely remove and what not. This is not a temporary solution. This is the final solution simply with a `Wanda` suffix at the end, as a reminder that the command is used with the wanda version.

Reply:
Let's go with what we have, as said, I am fine with it :rocket: 

For me to better understand: we're gonna end up with only a command named `Trento.Domain.Commands.CompleteChecksExecution` (without Wanda) which will have these fields
```elixir
defcommand do
  field :cluster_id, Ecto.UUID
  field :health, Ecto.Enum, values: Health.values()
end
```
instead of these
```elixir
defcommand do
  field :cluster_id, Ecto.UUID
  embeds_many :hosts_executions, HostExecution
end
```

Got it right?

Reply:
Yes